assets/components/manageContributionCta/manageContributionCta.scss

```diff
@@ -0,0 +1,29 @@
+.component-manage-contribution-cta {
+
+  .component-cta-link--manage-contribution {
```

Comment:
According to the CSS guidelines, we are trying to avoid nesting in the CSS files. Is it possible to do it in this case?